src/GapSkipper.js

```diff
@@ -0,0 +1,85 @@
+import videojs from 'video.js';
```

Comment:
I also think we wanted to use this more like a plugin for video.js https://github.com/videojs/video.js/blob/master/docs/guides/plugins.md